idaes/core/solvers/petsc.py

```diff
@@ -428,7 +428,7 @@ def petsc_dae_by_time_element(
     symbolic_solver_labels=True,
     between=None,
     interpolate=True,
-    calculate_derivatives=True,
+    calculate_derivatives=False,
```

Comment:
Because 1) Due to the indent issue, derivatives weren't being calculated by default, so this is consistent with the behavior before this fix and 2) I wanted the user to understand how `calculate_derivatives` worked (by solving the Pyomo discretization equations for derivative values) before having derivative values populated (lest they think the derivative values were, for example, taken straight from PETSc).